libs/core/langchain_core/indexing/api.py

```diff
@@ -215,10 +216,6 @@ def index(
      are not able to specify the uid of the document.
 
     IMPORTANT:
-       * if auto_cleanup is set to True, the loader should be returning
```

Comment:
BTW, `auto_cleanup` doesn't exist now.

Reply:
This looks incorrectly documented due to a refactor in names.

`auto_cleanup` sounds like `full` mode?

Reply:
Agree. I will change it to `full` later.